sdk/src/main/java/nl/rabobank/gict/payments_savings/omnikassa_frontend/sdk/model/response/OrderStatusResponse.java

```diff
@@ -0,0 +1,71 @@
+package nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.response;
+
+import kong.unirest.json.JSONObject;
+
+import nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.Money;
+
+import java.util.ArrayList;
+import java.util.Calendar;
```

Comment:
Remove unused imports. Calendar is not been used anywhere.

Reply:
Removed.